### PR TITLE
Added missing lexicon for 'refresh_media_sources'

### PR DESCRIPTION
**Pull request by Ruslan-Aleev**
### What does it do?
Added the missing lexicon for `refresh_media_sources` in the console when clearing the cache.

refresh_ms

### Related issue(s)/PR(s)
N/A

### CODE REVIEW

**Review by Jako — APPROVED** [review]

File: core/lexicon/en/default.inc.php [hunk 1]
```diff
@@ -374,6 +374,7 @@
 $_lang['refresh_default'] = 'Clearing the default cache';
 $_lang['refresh_failure'] = 'Refresh failed! (NOTE: this cache partition may just be empty)';
 $_lang['refresh_lexicon_topics'] = 'Clearing the lexicon topics cache';
+$_lang['refresh_media_sources'] = 'Clearing the media sources cache';
```
Comment (JoshuaLuckers) [inline core/lexicon/en/default.inc.php]:
```suggestion
$_lang['refresh_media_sources'] = 'Clearing the Media Sources cache';
```

**Review by JoshuaLuckers — APPROVED** [review]

**Comment by Ruslan-Aleev** [comment]
@JoshuaLuckers A 'lexicon topics' do not need to change to 'Lexicon Topics'?

**Comment by JoshuaLuckers** [comment]
Not sure if we should change it here or via CrowdIn. But for this PR leave it as it is 😄.

**Comment by Jako** [comment]
Yes, and 'Menu' should be changed too. 

The English lexicon is changed on GitHub, the other ones on Crowdin.

**Pull request merged**